.github/workflows/pythontest-linux.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         # Emit pickled gsheets testing credentials from CI secret
         python -c "import pickle; import base64; import os; tok=pickle.loads(base64.b64decode(os.getenv('GSHEETS_TOKEN'))); tfile = open('token.pickle', 'wb'); pickle.dump(tok, tfile)"
         pip install pytest
-        pytest .
+        pytest --doctest-modules icubam/
```

Comment:
Ideally this option should go in `setup.cfg` when it's added.

Reply:
I think we're not set up with a good versioning system.  We started putting versions in the config.toml but that was a bad idea, I think bumping the setup.py version every time we deploy could work.  What would you suggest?

Reply:
Ok, sorry I hadn't read through the json.  I think it's better to not print any information about beds etc. but to just run those calls inside try/catch blocks to make sure nothing bad is happening.  Ideally we'd have the git hash and package version.  Currently the version is accessible from the config with self.config.version, but I think that ought to be moved to icubam.__version__ or something easier to maintain.

Reply:
Thanks a lot Roman for the PR. I agree wih Gabriel (maybe we did not communicate clearly at first, sorry about that). but just having git commit hash, maybe commit tag and version would be perfect. Which simplifies a lot the PR. Then the route could be something like version/ ?

Reply:
Thanks for the feedback! Sure will make the changes -- that will simplify things indeed.

Reply:
hi roman, it s nice, but i am not sure this route should be only accessible to authenticated people. That is why i suggest to simplify it even more with no db access. Sorry to make you erase what you wrote. But please double check this statement with Romain before.
Thanks

Reply:
i  mean errors is not a bad idea at all... but double check how this would be accessed first